promscale/installation/upgrade.md

```diff
@@ -0,0 +1,70 @@
+# Upgrade Promscale
+Promscale consists of the Promscale Connector, and the Promscale PostgreSQL
+extension. When you upgrade your Promscale installation, you need to check both
+components. This section contains important information about upgrading.
+
+The process for updating your Promscale installation is:
```

Comment:
I think we need a process for when only the extension needs to be upgraded and maybe a separate process for when PostgreSQL and/or TimescaleDB need to be upgraded. Ideally we can point to Postgres and/or TimescaleDB documentation for those upgrades.